probcal/models/faithful_gaussian_nn.py

```diff
@@ -28,8 +28,8 @@ def __init__(
         self,
         backbone_type: Type[Backbone],
         backbone_kwargs: dict,
-        optim_type: OptimizerType,
-        optim_kwargs: dict,
+        optim_type: OptimizerType = None,
+        optim_kwargs: dict = None,
```

Comment:
Can we update the types on these so they're typed optionally? It'll look like

```
optim_type: OptimizerType | None = None,
optim_kwargs: dict | None = None,
```

Reply:
Just added this for faithful gaussian, natural gaussian, and negative binomial